engines/bastion_katello/test/debs/debs.controller.test.js

```diff
@@ -0,0 +1,110 @@
+describe('Controller: DebsController', function() {
+    var $scope,
+        $location,
+        $controller,
+        dependencies,
+        Deb,
+        //Task,
+        Nutupane;
+
+    beforeEach(module('Bastion.debs', 'Bastion.test-mocks'));
+
+    beforeEach(function() {
+        Nutupane = function() {
+            this.table = {
+                params: {},
+                showColumns: function() {}
+            };
+            this.get = function() {};
+            this.setParams = function (params) {};
+            this.getParams = function (params) { return {}; };
+            this.refresh = function () {};
+            this.getAllSelectedResults = function () {};
+        };
+        Errata = {};
+
+/*
```

Comment:
Is this TODO accurate?

Reply:
Hm, I severily lack experience with Angular and these tests. I guess what I did was to copy most of it from RPM and added TODOs for those that I could not get running :thinking: 

I'll try to have a second look but help would be appreciated :grin: